src/views/LocusGeneDiseaseView.vue

```diff
@@ -66,6 +67,14 @@ export default {
         .then((responseJson) => {
           this.isDataLoading = false;
           this.locusGeneDiseaseData = responseJson;
+          if (
+            this.locusGeneDiseaseData.curators &&
+            this.locusGeneDiseaseData.curators.length > 1
+          ) {
+            this.curatorString = this.locusGeneDiseaseData.curators.join(",");
+          } else {
+            this.curatorString = this.locusGeneDiseaseData.curators[0];
+          }
```

Comment:
locusGeneDiseaseData.curators.join(",") will be able to handle the case where length is 1....so I think in terms of code simplicity, this code block can be taken care within line 1023 itself